### ISSUE TITLE
Monk level 17 not completing

### ISSUE THREAD

**Issue by raindropworks** [rwtema/modjam2018#68]
If I'm reading the config file right, for level 17 you need to fall 40 blocks and land on a hay bale, but having done it from over 100 blocks up, one of my modpack users isn't getting the achievment for it.

No changes have been made to the mod since installation, so I'm not sure what's going on

**Comment by darthvader45** [comment]
Are they wearing armor?  That might prevent it from working.

**Comment by jasoncollege24** [comment]
I'm having this same problem, no armor, no baubles, and not holding anything. I jump off a block 63 blocks higher than the hay bale... i don't die, but I don't get the advancement, or it's skill

**Comment by darthvader45** [comment]
Weird, I've had it be a bit inconsistent.  Keep trying, had to do it a few times to get it right.  Once I have the right height I'll post it here by editing my comment.

**Comment by jasoncollege24** [comment]
i tried repeatedly over the course of an hour. In the end, I was forced to use the command "/monk setlevel 16" (which is the correct level, not 17).

To be fair, I did import that particular world from an earlier version of the mod pack (direwolf20 1.12.2 v2.5.0 imported to into v2.9.0).

In the original modpack, I had removed the monk mod, then later decided I wanted it back, but didn't want to redo all the other work I'd done  and didn't know how to reinstall the correct version of the mod. It's possible I corrupted something.

Edit: I went back to the original instance, and reinstalled the Monk mod. I progressed all the way through, and when I got to this, I got stuck again. This time, the key was to remove ALL gear, including baubles.

**Comment by darthvader45** [comment]
Yep, apparently it considers baubles as armor.  Make sure you aren't wearing anything at all.  Then it works.
